arbor/profile/profiler_macro.hpp

```diff
@@ -4,6 +4,9 @@
 
 #ifdef ARB_HAVE_PROFILING
 
+    // initialize profiler
+    #define PI(ts) arb::profile::profiler_initialize(ts)
```

Comment:
The convenience macros `PE` and `PL` are defined to make them easier to type and less invasive in code.
I don't think a macro is needed for initializing the profiler, because it will only be done once, and it is nice to be clear/explicit about it.